recipes/metagem/post-link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --user memote carveme smetana
```

Comment:
you can move this one-liner directly into the meta.yml as `script`

Reply:
thanks!